doc/script_commands.txt

```diff
@@ -5317,6 +5317,27 @@ Check getitem2() to understand its expanded parameters.
 
 ---------------------------------------
 
+*delitemidx(<index>{, <account id>})
+
+This command will remove the item at the given inventory index.
+
+Even if the amount found is more than 1, it will remove all of the item.
```

Comment:
this is hell, no, don't do this
we should allow delete an amount specify by the script command